extensions/bond/sawtooth_bond/updates/libor.py

```diff
@@ -47,7 +47,8 @@ def __init__(self, date, rates, signature=None):
                 OneYear: The one-month LIBOR
             signature: A signature over the LIBOR data (date and rates)
```

Comment:
Sorry.  I should have been more clear.  You added the public_key parameter, but you did not document it.